config/application.example.properties

```diff
@@ -4,11 +4,16 @@
 ##################################################################################################
 
 # CH Connect Configuration
-connect.issuerUri=https://connect.ch.tudelft.nl/
-connect.clientUri=http://localhost:9000/
+wisvch.connect.issuerUri=https://connect.ch.tudelft.nl/
+wisvch.connect.clientUri=http://localhost:8080/events
+
+wisvch.connect.registered-client.client-name=CH Events
+wisvch.connect.registered-client.client-id=events
+wisvch.connect.registered-client.client-secret=<ASK_AN_EVENTS_DEV_FOR_KEY>
```

Comment:
In fact, it should work just fine if you're not using the `prod` Spring profile.